src/components/cc-toggle/cc-toggle.js

```diff
@@ -94,6 +94,7 @@ export class CcToggle extends LitElement {
     this.value = null;
   }
 
+  /** @param {Event & { target: HTMLInputElement }} e */
```

Comment:
nitpick: can be simplified by `EventWithTarget<HTMLInputElement>` (@see `events.types.d.ts`)

Reply:
I didn't remember you had merged this one, this is great thanks! :heart: